### ISSUE TITLE
Training Logs of Successfull Runs

### ISSUE THREAD

**Issue by mbreuss** [allenzren/open-pi-zero#3]
HI @allenzren,

Thank you for your great work! I am very interested to see some training logs of your successful runs. It is possible to share these?
I also noticed that you are working with 8 grad accumulate steps to reach very large batch sizes. Did you also test these trainings without the accumulation steps or have any insights you can share on that? 

Thanks a lot!

**Comment by allenzren** [comment]
Hi Moritz, thanks for the message! Huge fan of your work.

I honestly haven't tuned the training setup as much. I did try global batch size 256 or more than 1024, and the training curves (wall-clock time vs. training loss) were similar; not sure about the eval though. I think both OpenVLA and Octo use batch size 2048, so I thought 1024 is not that large.

My training logs were not in great shape because for some runs I had to resume training for than once. This one for Bridge-Beta is good though: https://api.wandb.ai/links/allenzren/6q4xihqu The training was very stable. Is there anything you are particularly concerned with?

I am re-training the models again without resuming now and they should be done tomorrow. I can share the full logs and new evals then.

**Comment by allenzren** [comment]
Btw, I just came across your OXE dataloader https://github.com/mbreuss/oxe_torch_dataloader/tree/main, looks great! Is it more efficient than the Octo TFDS loaders?

**Comment by mbreuss** [comment]
Thanks for sharing the logs and your experiences!
I am working on a similar training setup so having something to compare against is very helpful! If you have some time after rss deadline I would be interested to hear your experiences in a short call if you have some time for that. 

> Btw, I just came across your OXE dataloader https://github.com/mbreuss/oxe_torch_dataloader/tree/main, looks great! Is it more efficient than the Octo TFDS loaders?

My former student developed that. It has some changes for our older experiments and models and some torch changes for the high level classes that should make it a bit faster. Given the internal changes not sure how easy it is to extract the relevant features only. I can give it a shot to implement it here and make a comparison end of January.
